src/renderer/widgets/CreateWallet/ui/MultisigWallet/FlexibleMultisigWallet.tsx

```diff
@@ -59,33 +59,29 @@ export const MultiChainMultisigWallet = ({ isOpen, onClose, onComplete, onBack }
   }, [isOpen]);
 
   useEffect(() => {
-    createMultisigWalletModel.events.signatoriesChanged(signatories);
-  }, [signatories.length]);
-
-  useEffect(() => {
-    createMultisigWalletModel.events.callbacksChanged({ onComplete });
+    flowModel.events.callbacksChanged({ onComplete });
   }, [onComplete]);
 
-  const goToPrevStep = () => {
-    if (activeStep === Step.INIT) {
-      onBack();
-    } else {
-      setActiveStep((prev) => prev - 1);
-    }
-  };
+  // const goToPrevStep = () => {
+  //   if (activeStep === Step.INIT) {
+  //     onBack();
+  //   } else {
+  //     setActiveStep((prev) => prev - 1);
+  //   }
+  // };
 
   const closeMultisigModal = (params: { complete?: boolean; closeAll?: boolean } = { closeAll: true }) => {
     toggleIsModalOpen();
 
     setTimeout(params?.complete ? onComplete : params?.closeAll ? onClose : noop, DEFAULT_TRANSITION);
   };
 
-  const submitHandler = (args: any) => {
-    toggleResultModal();
-    setName(args.name);
+  // const submitHandler = (args: any) => {
```

Comment:
remove it?

Reply:
same here, this is for the flexible multisig